Classes/SDK/PXLPhoto.swift

```diff
@@ -128,7 +128,9 @@ public struct PXLPhoto: Equatable {
     }
 
     public var sourceIconImage: UIImage? {
-        let bundle = Bundle(for: PXLAlbum.self)
+        // test it in Cocoapods
+        // let bundle = Bundle(for: PXLAlbum.self)
+        let bundle = Bundle.module
```

Comment:
does this need the preprocessor macro like the others?
```suggestion
        #if SWIFT_PACKAGE
        let bundle = Bundle.module
        # else
        let bundle = Bundle(for: PXLAbum.self)
        #endif
```